### PR TITLE
Lost changes to P1754R1 (LWG Motion 11)

### PR DESCRIPTION
**Pull request by burblebee**
These fixes from branch motions-2019-07-lwg-11 were lost in the new branch motions-2019-07-lwg-11-rs.  The 2nd commit in this PR encodes additional but unconventional uses of concept names with \libconcept, but more work may still be needed - see related inline comments in PR #3149.

Update 2/25/21: Most of the changes in the original patch had already been fixed in master since this PR was created.  The rebased branch has been updated to include the few changes that remain in master.

### CODE REVIEW

**Comment by burblebee** [comment]
The build is timing out with "Timeout waiting for network availability."

**Comment by jensmaurer** [comment]
Please rebase and fix the conflicts, then force-push.

**Comment by tkoeppe** [comment]
I think this is obsolete now, @burblebee?

**Comment by burblebee** [comment]
@tkoeppe, no these changes were absolutely needed. I didn't realize they hadn't been applied yet.
I still have the branch thankfully - will see what still needs to be applied after we get thru the current motions.

**Comment by tkoeppe** [comment]
Aha, thank you, sorry about that! Could you please open a regular pull request for those changes then? Thanks!

**Comment by burblebee** [comment]
> Aha, thank you, sorry about that! Could you please open a regular pull request for those changes then? (Or indeed reopen and update this one?) Thanks!

Is this not a regular PR?  Yes, will update and rebase after the current motions are applied ( I suspect many of these changes have been found and fixed already).

**Comment by tkoeppe** [comment]
Sorry, yes, this is of course a regular PR, I failed to say that I was implicitly thinking "from a branch on your own fork", as opposed to adding PR branches to this repo. But let's leave this as is now, once it's rebased and dealt with, we'll delete the branch. Thanks!

File: source/support.tex [hunk 1]
```diff
@@ -4188,9 +4188,9 @@
 
   // \ref{cmp.concept}, concept \libconcept{three_way_comparable}
   template<class T, class Cat = partial_ordering>
-    concept three_way_comparable = @\seebelow@;
+    concept @\deflibconcept{three_way_comparable}@ = @\seebelow@;
   template<class T, class U, class Cat = partial_ordering>
-    concept three_way_comparable_with = @\seebelow@;
+    concept @\deflibconcept{three_way_comparable_with}@ = @\seebelow@;
```
Comment (jensmaurer) [inline source/support.tex:4193]: This should not be `\deflibconcept` here; the `\deflibconept` for these is further down, with the actual (semantic) definition. Make it a `\libconcept`, please.

File: source/utilities.tex [hunk 2]
```diff
@@ -7093,9 +7093,9 @@
   template<class T1, class D1, class T2, class D2>
     requires @\libconcept{three_way_comparable_with}@<typename unique_ptr<T1, D1>::pointer,
                                        typename unique_ptr<T2, D2>::pointer>
-    compare_three_way_result_t<typename unique_ptr<T1, D1>::pointer,
-                               typename unique_ptr<T2, D2>::pointer>
-      operator<=>(const unique_ptr<T1, D1>& x, const unique_ptr<T2, D2>& y);
+      compare_three_way_result_t<typename unique_ptr<T1, D1>::pointer,
+                                 typename unique_ptr<T2, D2>::pointer>
+        operator<=>(const unique_ptr<T1, D1>& x, const unique_ptr<T2, D2>& y);
```
Comment (jensmaurer) [inline source/utilities.tex:7098]: This should be dropped, I believe. "requires" does not cause additional two-space indentation.
(same for the rest here)

**Review by jensmaurer — CHANGES_REQUESTED** [review]

File: source/support.tex (hunk 1 above)
Comment (burblebee) [reply source/support.tex]: right - I caught that also and fixed.  You're too fast for me! :)

File: source/utilities.tex (hunk 2 above)
Comment (burblebee) [reply source/utilities.tex]: We're inconsistent about this.  If we don't want extra indents after requires, then we should fix the places that do indent.  I'm fine either way, although I have a slight preference to not indent also.  I've removed the extra indents from this patch for now.

**Review by jensmaurer — APPROVED** [review]

File: source/support.tex (hunk 1 above)
Comment (burblebee) [reply source/support.tex]: Oh, and as far as making it a \libconcept, we don't seem to do that in general for the declaration.  Should we? In the synopsis, we (typically) only use \libconcept in requires clauses and parameters. We do use \exposconcept for the \expos case however (which makes sense).  See:
`grep "^[ ]*concept .*=" *.tex |grep -v deflibcon |grep -v defexpos |grep -v seebelow`

Comment (jensmaurer) [reply source/support.tex]: It's not really useful to add "use" index entries pointing to the synopsis of a definition if we have a proper `\itemdecl` definition in the vicinity.

Comment (burblebee) [reply source/support.tex]: agreed

**Pull request merged**